packages/erebos-api-bzz-base/src/index.js

```diff
@@ -18,6 +28,15 @@ export default class BaseBzz {
     }).then(res => res.text())
   }
 
+  download(hash: string, path?: string = ''): Promise<*> {
+    const content_path = path === '' ? '' : `/${path}`
```

Comment:
camelCase

Reply:
Elixir-lang influence is leaking :sweat_smile: